.changeset/famous-experts-begin.md

```diff
@@ -0,0 +1,9 @@
+---
+'@clerk/clerk-js': patch
```

Comment:
🚫 We shouldn't have different entries for `clerk-js` (minor vs patch) 

Also, all of the changes on this PR should be placed on minor versions. The structure could be the following:
- Changeset for `@clerk/backend` giving more context around the version update.
- Changeset for `@clerk/types` & `@clerk/clerk-js` with adding support for `enterprise_sso` strategy and on the user settings. Give more context on when `enterprise_sso` resolves to `true` in comparison to `saml`.

Reply:
Makes sense to have a separate changeset for each package. The versions look correct to me now 👍🏻